pysindy/optimizers/constrained_stlsq.py

```diff
@@ -0,0 +1,169 @@
+import warnings
+
+import numpy as np
+from sklearn.exceptions import ConvergenceWarning
+from sklearn.linear_model import ridge_regression
+
+from pysindy.optimizers import BaseOptimizer
+
+
+class ConstrainedSTLSQ(BaseOptimizer):
+    """
+    Sequentially thresholded least squares algorithm.
+
+    Attempts to minimize the objective function
+    ||y - Xw||^2_2 + alpha * ||w||^2_2 by iteratively performing
```

Comment:
Could you apply the correct sphinx formatting here please?

Reply:
I'll fix this.